pyartcd/pyartcd/pipelines/build_fbc.py

```
@@ -118,31 +120,22 @@ async def _rebase_and_build(self, release: str, commit_message: str):
-            if not final_kubeconfig:
-                available_env_vars = list(GROUP_KUBECONFIG_MAP.values())
-                raise ValueError(
-                    f"Kubeconfig required for Konflux builds. Provide --kubeconfig parameter or set one of: {', '.join(available_env_vars)}"
-                )
+        # Use kubeconfig from CLI parameter or product-specific environment variable
+        final_kubeconfig = resolve_konflux_kubeconfig_by_product(product, self.kubeconfig)
+        if not final_kubeconfig:
+            available_env_vars = list(PRODUCT_KUBECONFIG_MAP.values())
+            raise ValueError(
+                f"Kubeconfig required for Konflux builds. Provide --kubeconfig parameter or set one of: {', '.join(available_env_vars)}"
+            )
```

Ruff (0.14.3)

Avoid specifying long messages outside the exception class

(TRY003)